src/WorldStorage/WSSAnvil.cpp

```diff
@@ -2662,15 +2664,7 @@ void cWSSAnvil::LoadSilverfishFromNBT(cEntityList & a_Entities, const cParsedNBT
 
 void cWSSAnvil::LoadSkeletonFromNBT(cEntityList & a_Entities, const cParsedNBT & a_NBT, int a_TagIdx)
 {
-	int TypeIdx = a_NBT.FindChildByName(a_TagIdx, "SkeletonType");
-	if (TypeIdx < 0)
-	{
-		return;
-	}
-
-	bool Type = ((a_NBT.GetByte(TypeIdx) == 1) ? true : false);
-
-	std::unique_ptr<cSkeleton> Monster = cpp14::make_unique<cSkeleton>(Type);
```

Comment:
Did the NBT format for skeletons / wither skeletons change? Shouldn't we still support both formats?

Reply:
We should probably keep this for legacy support.